pandas/tests/windows/test_expanding.py

```diff
@@ -0,0 +1,111 @@
+from datetime import datetime
+
+import numpy as np
+import pytest
+from numpy.random import randn
+
+import pandas as pd
+import pandas.core.window as rwindow
+import pandas.util.testing as tm
+from pandas import DataFrame, Series, bdate_range
+from pandas.errors import UnsupportedFunctionCall
+
+N, K = 100, 10
+
+
+def assert_equal(left, right):
+    if isinstance(left, Series):
+        tm.assert_series_equal(left, right)
+    else:
+        tm.assert_frame_equal(left, right)
+
+
+class Base(object):
+
+    _nan_locs = np.arange(20, 40)
+    _inf_locs = np.array([])
+
+    def _create_data(self):
+        arr = randn(N)
+        arr[self._nan_locs] = np.NaN
+
+        self.arr = arr
+        self.rng = bdate_range(datetime(2009, 1, 1), periods=N)
+        self.series = Series(arr.copy(), index=self.rng)
+        self.frame = DataFrame(randn(N, K), index=self.rng,
+                               columns=np.arange(K))
+
+
+class TestExpanding(Base):
+
+    def setup_method(self, method):
+        self._create_data()
+
+    def test_doc_string(self):
+
+        df = DataFrame({'B': [0, 1, 2, np.nan, 4]})
+        df
+        df.expanding(2).sum()
+
+    def test_constructor(self):
+        # GH 12669
+
+        for o in [self.series, self.frame]:
+            c = o.expanding
+
+            # valid
+            c(min_periods=1)
+            c(min_periods=1, center=True)
+            c(min_periods=1, center=False)
+
+            # not valid
+            for w in [2., 'foo', np.array([2])]:
+                with pytest.raises(ValueError):
+                    c(min_periods=w)
+                with pytest.raises(ValueError):
+                    c(min_periods=1, center=w)
+
+    def test_numpy_compat(self):
```

Comment:
parameterize this

Reply:
Can you hlep me? Here I need use a decorator `@pytest.mark.parametrize`?